cli/.js/src/main/scala/aqua/run/Runner.scala

```diff
@@ -169,7 +169,7 @@ class Runner(
         case (Some(s), _: BoxType) if s.function.arg == js.undefined => Validated.validNec(s :: Nil)
         case (Some(s), _) if s.function.arg == js.undefined =>
           Validated.invalidNec(
-            s"Argument '$n' is undefined, but it's type '$argType' cannot be undefined."
+            s"Argument '$n' is undefined, but it must be of '$argType' type"
```

Comment:
What is 'undefined' in Aqua? It's a terminology from JS. I'd say "Argument '$n' is missing"